nibabel/cmdline/diff.py

```diff
@@ -125,6 +140,76 @@ def get_data_diff(files):
     return md5sums
 
 
+def get_data_diff(files, max_abs=0, max_rel=0):
+    """Get difference between data
+
+    Parameters
+    ----------
+    files: list of (str or ndarray)
+      If list of strings is provided -- they must be existing file names
+    max_abs: float, optional
+      Maximal absolute difference to tolerate.
+    max_rel: float, optional
+      Maximal relative (`abs(diff)/mean(diff)`) difference to tolerate.
+      If `max_abs` is specified, then those data points with lesser than that
+      absolute difference, are not considered for relative difference testing
+
+    Returns
+    -------
+    OrderedDict
+        str: absolute and relative differences of each file, given as float
+    """
+    # we are doomed to keep them in RAM now
+    data = [f if isinstance(f, np.ndarray) else nib.load(f).get_data() for f in files]
```

Comment:
Given that `get_data()` is on its way out, I would suggest using either `dataobj.get_unscaled()` or `get_fdata(np.float32)` here, depending on whether you're interested in on-disk values or (more likely) scaled values.